tools/convert_votenet_checkpoints.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import torch
+from mmcv import Config
+from mmcv.runner import load_state_dict
+
+from mmdet3d.models import build_detector
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='MMDet test (and eval) a model')
+    parser.add_argument('config', help='test config file path')
+    parser.add_argument('checkpoint', help='checkpoint file')
+    parser.add_argument('--out', help='path of the output checkpoint file')
+    parser.add_argument(
+        '--model',
+        choices=['sunrgbd', 'scannet'],
+        default='sunrgbd',
+        help='type of the model')
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = parse_args()
+    cfg = Config.fromfile(args.config)
+
+    if args.model == 'scannet':
+        NUM_CLASSES = 18
+    else:
+        NUM_CLASSES = 10
+
+    EXTRACT_KEYS = {
+        'bbox_head.conv_pred.conv_cls.weight':
+        ('bbox_head.conv_pred.conv_out.weight', [(0, 2), (-NUM_CLASSES, -1)]),
+        'bbox_head.conv_pred.conv_cls.bias':
+        ('bbox_head.conv_pred.conv_out.bias', [(0, 2), (-NUM_CLASSES, -1)]),
+        'bbox_head.conv_pred.conv_reg.weight':
+        ('bbox_head.conv_pred.conv_out.weight', [(2, -NUM_CLASSES)]),
+        'bbox_head.conv_pred.conv_reg.bias':
+        ('bbox_head.conv_pred.conv_out.bias', [(2, -NUM_CLASSES)])
+    }
+
+    RENAME_KEYS = {
+        'bbox_head.conv_pred.shared_convs.layer0.conv.weight':
+        'bbox_head.conv_pred.0.conv.weight',
+        'bbox_head.conv_pred.shared_convs.layer0.conv.bias':
+        'bbox_head.conv_pred.0.conv.bias',
```

Comment:
We can directly replace `bbox_head.conv_pred` by `bbox_head.conv_pred.shared_convs.layer`?